lib/metrics.py

```diff
@@ -285,6 +288,23 @@ def _get_number_of_files(self):
                 raise Exception("Unexpected result from database query")
             return int(rows[0][0])
 
+    def _get_size_of_files(self):
+        conn = self._get_db_conn()
+        with conn.cursor() as cursor:
+            try:
+                cursor.execute(
+                    "SELECT sum(size) from system$filedocument WHERE hascontents=true;"
+                )
+                rows = cursor.fetchall()
+                if len(rows) == 0:
+                    return 0
+                return int(rows[0][0])
+            except Exception as e:
+                # We ignore errors here, as the information is not available for
+                # older mendix versions
+                logger.debug("METRICS: Error retrieving file sizes", e)
```

Comment:
Since we are only logging this at `DEBUG` level, then it would be good to include the traceback, replace `e` with `exc_info=True`.